### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ScreenReader:Narrator announces each widget cell present on the page as "Group" and does not give any other information about what exactly is present on the page.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4205]
Environment Details:
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 70.0.3538.77 (Official Build) (64-bit)
Operating System:Windows10

Steps to reproduce:
1.Login with the appropriate credentials to azure notebook account.    

2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library

3.Navigate to run button & hit enter --> Navigate to Creation and Deployment of an Azure ML Web Service & hit enter.
4. Navigate to Code Cell block-->Table

Actual
Narrator announces each widget cell present on the page as "Group" and does not give any other information about what exactly is present on the page.

Expected
Narrator should announces each widget cell present on the page with proper information about the  content that is displayed on the page or the user is navigating through.
It should give some cell reference to the end user.
 
User Impact
End User will be unaware of the information present on the screen if that is not conveyed by the screen reader.

Recommendation
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

MAS Reference
https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D

[A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ScreenReader.zip](https://github.com/jupyter/notebook/files/2576130/A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ScreenReader.zip)

**Comment by ailsaleen** [comment]
Hello @AccessibilityTestingTeam-TCS and @minrk , I'm a PM on the Azure ML team. We tested this bug as it is listed as an outstanding issue for our product's accessibility and believe it no longer repros. Can we close this issue?

**Comment by ailsaleen** [comment]
Thank you! :)
